.pipelines/templates/install-helm.yml

```diff
@@ -0,0 +1,13 @@
+parameters:
+  version: v3.5.2
+
+steps:
+  - bash: |
+      curl -LO https://get.helm.sh/helm-${{ parameters.version }}-linux-amd64.zip
+      unzip -o helm-${{ parameters.version }}-linux-amd64.zip
```

Comment:
Why not install specific version of helm with the [install script](https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3)?